src/MainView.js

```diff
@@ -54,22 +110,32 @@ export const Visualizer = React.createClass({
               />
               Visualizer
             </div>
-            <ControlPanel className={this.state.menuVisible ? style.menu : style.hiddenMenu} />
+            <ControlPanel
+              className={this.state.menuVisible ? style.menu : style.hiddenMenu}
+              resetCamera={this.resetCamera}
+            />
           </div>
           <div className={style.buttons}>
             <TimeController />
             <i
               className={style.resetCameraButton}
-              onClick={this.props.resetCamera}
+              onClick={this.resetCamera}
             />
           </div>
         </div>
-        <VtkRenderer
-          ref={c => (this.renderer = c)}
+        <Renderer
+          ref={(c) => { this.renderer = c; }}
```

Comment:
I guess the goal of the `()` instead of `{}` was to make that ref call smaller. But doesn't matter.